scripts/deploy.py

```diff
@@ -59,15 +59,15 @@ def main():
     token = Token.at(get_address("ERC20 Token"))
 
     if use_proxy:
-        gov_default = dev.address
+        gov_default = (
+            "0x16388463d60FFE0661Cf7F1f31a7D658aC790ff7"  # strategist msig, no ENS
```

Comment:
can the strategist msig be added to ENS? e.g. `brains.ychad.eth`

Reply:
It's on Klim's long list already.